thirdparty/fluidsynth/fluidsynth-2.1.4/src/external/portable_endian.h

```diff
@@ -52,9 +52,9 @@
 
 #	include <sys/endian.h>
 
-#elif defined(__NetBSD__) || defined(__FreeBSD__) || defined(__DragonFly__)
+#elif defined(__NetBSD__) || defined(__DragonFly__)
 
-#	include <sys/endian.h>
+#x	include <sys/endian.h>

```

Comment:
It looks like the `x` symbol is superfluous here

Reply:
Yes that was a typo. I have refactored the portable_endian.h a litte